HW0/P5/P5.txt

```diff
@@ -1 +1,327 @@
-# Your answers here.
+###############################
+
+# CS 205 Fall 2015 Homework 0 Problem 5
+# Submitted by Kendrick Lo (Harvard ID: 70984997)
+# Github username: ppgmg
+
+Summary
+-------
+
+1. 255 seconds
+2. 34 seconds
+3. 8 seconds
+4. log N seconds
+5. 15 seconds
+6. 272 seconds, 143 seconds
```

Comment:
We are not concerned about the offset of (+1/-1 seconds). The key is to realize the log(N) trend. The discussion given is very thorough, great work!